### ISSUE TITLE
java.lang.NoSuchMethodError on launch and when a command is sent in discord

### ISSUE THREAD

**Issue by lmore377** [Awakened-Redstone/AutoWhitelist#18]
### Version information

1.1.2

### Expected Behavior

The mod should load without errors and properly respond to commands in discord

### Actual Behavior

The mod spits out an error on start and when a command is sent in discord.

### Reproduction Steps

1. Setup a fabric server with Minecraft version 1.19,4
2. Configure the mod like normal
3. Launch the server (an error is printed to console)
4. Try to run any commands thru discord (an error is printed to console)

### Java version

Java 19 (also affects Java 17)

### Additional information

Happens with the modrinth and compat versions. Minecraft version is 1.19.4. This occurs even when this mod, fabric-api and fabric-language-kotlin are the only installed mods.

Here's the full error from launch: 
```
java.lang.NoSuchMethodError: 'java.lang.String net.minecraft.class_2477.method_4679(java.lang.String)'
        at com.awakenedredstone.autowhitelist.discord.api.text.TranslatableText.updateTranslations(TranslatableText.java:49) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.TranslatableText.visitSelf(TranslatableText.java:146) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.Text.visit(Text.java:78) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.StringVisitable.getString(StringVisitable.java:50) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.Text.getString(Text.java:33) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.Bot.run(Bot.java:118) ~[autowhitelist-1.1.2.jar:?]
```

and the error generated when sending `np!help` in discord:
```
[00:56:58] [JDA MainWS-ReadThread/ERROR]: Couldn't access annotated EventListener method
java.lang.reflect.InvocationTargetException: null
        at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:119) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
        at net.dv8tion.jda.api.hooks.AnnotatedEventManager.lambda$handle$0(AnnotatedEventManager.java:95) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) ~[?:?]
        at net.dv8tion.jda.api.hooks.AnnotatedEventManager.lambda$handle$1(AnnotatedEventManager.java:90) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at java.util.concurrent.ConcurrentHashMap.forEach(ConcurrentHashMap.java:1603) ~[?:?]
        at net.dv8tion.jda.api.hooks.AnnotatedEventManager.handle(AnnotatedEventManager.java:90) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:88) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:70) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:160) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.handle.MessageCreateHandler.handleInternally(MessageCreateHandler.java:123) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:36) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:952) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:839) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:817) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:991) ~[net_dv8tion_jda-4.4.0_352-525c9f524e1d024d.jar:?]
        at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
        at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
        at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
        at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
        at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45) ~[com_neovisionaries_nv-websocket-client-2.14-2dec4abe300669c3.jar:?]
Caused by: java.lang.NoSuchMethodError: 'java.lang.String net.minecraft.class_2477.method_4679(java.lang.String)'
        at com.awakenedredstone.autowhitelist.discord.api.text.TranslatableText.updateTranslations(TranslatableText.java:49) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.TranslatableText.visitSelf(TranslatableText.java:146) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.Text.visit(Text.java:78) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.Text.visit(Text.java:91) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.StringVisitable.getString(StringVisitable.java:50) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.text.Text.getString(Text.java:33) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.BotHelper.sendFeedbackMessage(BotHelper.java:32) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.command.DiscordCommandSource.sendError(DiscordCommandSource.java:51) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.api.command.CommandManager.execute(CommandManager.java:77) ~[autowhitelist-1.1.2.jar:?]
        at com.awakenedredstone.autowhitelist.discord.events.GatewayEvents.onMessage(GatewayEvents.java:25) ~[autowhitelist-1.1.2.jar:?]
        at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
        ... 22 more
```

**Comment by Awakened-Redstone** [comment]
For 1.19.4 please use 1.1.4 or newer
I have published a new version for 1.19.4
https://modrinth.com/mod/autowhitelist/version/1.1.5

**Comment by lmore377** [comment]
Just tested and it works well. Thanks for the quick fix!
